ui/src/app/store/pod/types.ts

```diff
@@ -83,12 +88,13 @@ export type BasePod = Model & {
   pool: number;
   power_address: string;
   power_pass?: string;
+  // Only LXD pods have the project parameter.
+  project?: string;
   owners_count: number;
-  // The websocket does not include this value if the pod has no storage pools
-  storage_pools?: PodStoragePool[];
+  storage_pools: PodStoragePool[];
```

Comment:
This has been fixed on the api side to always return the parameter, but return an empty array if there are no storage pools detected